package.json

```diff
@@ -27,6 +27,7 @@
     "@clack/prompts": "0.7.0",
     "@sentry/cli": "^1.77.3",
     "@sentry/node": "^7.119.2",
+    "@sentry/sveltekit": "^9.1.0",
```

Comment:
any chance this is a leftover from an e2e test run?

Reply:
possible, will revert.